test_app/scripts/bootstrap.sh

```diff
@@ -25,7 +25,7 @@ then
     make postgres
 else
     echo "dab_postgres container is already running, will use that container"
-    python manage.py migrate --check; migrate_needed=$?
+    migrate_needed=$(echo $(python manage.py migrate --check > /dev/null 2> /dev/null; echo $?))
```

Comment:
This might be a cleaner fix.

Setting `-e` at the top is the reason we're bailing when the first command fails.

```suggestion
    set +e
    python manage.py migrate --check; migrate_needed=$?
    set -e
```

Reply:
Or we can just get rid of the `-e` at the top if we wanted.

Reply:
This does result in a fairly long traceback that ends in:

```
django.db.utils.OperationalError: connection failed: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
```

But verbosity isn't really my concern, it still comes up. So I'll commit this.

Reply:
well we don't need the output from the command here, so we can pipe it to >/dev/null (or 2>/dev/null if it's going to stderr, I'm not sure).

That would go before the semicolon, so:

```suggestion
    set +e
    python manage.py migrate --check 2> /dev/null; migrate_needed=$?
    set -e
```

I didn't test this - if it's still verbose then change the `2>` to just `>`

Reply:
Yeah, but we don't do that in any other cases. I'd worry about suppressing other errors. I've put in your first suggestion. That looks good to me.